src/node_i18n.cc

```diff
@@ -64,7 +79,122 @@ bool InitializeICUDirectory(const char* icu_data_path) {
   }
 }
 
+static int32_t ToUnicode(MaybeStackBuffer<char>* buf,
+                         const char* input,
+                         size_t length) {
+  UErrorCode status = U_ZERO_ERROR;
+  uint32_t options = UIDNA_DEFAULT;
+  options |= UIDNA_NONTRANSITIONAL_TO_UNICODE;
+  UIDNA* uidna = uidna_openUTS46(options, &status);
```

Comment:
technically, yeah. I haven't found a case where this line actually fails but it would be good house keeping